amcl/test/basic_localization_stage.xml

```diff
@@ -4,7 +4,7 @@ setting pose: 30.329 34.644 3.142
 <launch>
     <param name="/use_sim_time" value="true"/>
     <node name="rosbag" pkg="rosbag" type="play" 
-        args="-d 5 -r 1 --clock --hz 10 $(find amcl)/test/basic_localization_stage_indexed.bag"/>
+        args="-d 5 -r 5 --clock --hz 10 $(find amcl)/test/basic_localization_stage_indexed.bag"/>
```

Comment:
is this change accelerating playback by 5x?

Reply:
yes, it's making the test run faster.